src/pages/RFOX/hooks/helpers.ts

```diff
@@ -11,25 +11,22 @@ import type { AbiStakingInfo, PartialEpochMetadata, StakingInfo } from '../types
  * accounting on-chain.
  */
 export const calcEpochRewardForAccountRuneBaseUnit = (
-  epochEarningsForAccount: bigint,
-  epochMetadata: PartialEpochMetadata,
+  epochRewardUnitsForAccount: bigint,
+  epoch: PartialEpoch,
 ) => {
-  const secondsInEpoch: bigint = epochMetadata.endTimestamp - epochMetadata.startTimestamp + 1n
```

Comment:
This was incorrect for the current epoch because we're meant to be displaying the rewards SO FAR, not the total rewards for the whole month. All of the values used to calculate the current epoch pending rewards should be from the epoch start up to the current timestamp, including:

* affiliate revenue
* reward units for the account
* total reward units (which is based on `secondsInEpoch`)